utils/common/src/main/kotlin/DiskCache.kt

```diff
@@ -45,7 +45,7 @@ class DiskCache(
     /**
      * Duration in seconds that cache entries are valid.
      */
-    private val maxCacheEntryAgeInSeconds: Int
+    private val maxCacheEntryAgeInSeconds: Long
```

Comment:
Int was 68 years was it?

Reply:
I don't know, but the math using `maxCacheEntryAgeInSeconds` was using `Long` already (due to API requirements), so it anyway made sense to turn this into `Long`, too.